src/scheduler/node_info.c

```diff
@@ -2954,6 +2954,12 @@ eval_simple_selspec(status *policy, chunk *chk, node_info **pninfo_arr,
 		if (flags & EVAL_OKBREAK)
 			free_nodes(ninfo_arr);
 		return 0;
+	} else if (resresv->aoename != NULL) {
```

Comment:
Is this really an else if, or an if?  There is no harm making it an if, because the previous if statement returns.

Reply:
Done.